jdatetime/__init__.py

```diff
@@ -609,7 +615,7 @@ def strftime(self, format):
         format = format.replace("%W", str(self.weeknumber()))
 
         if '%x' in format:
-            format = format.replace("%x", self.strftime("%m/%d/%y"))
+            format = format.replace("%x", self.strftime("%Y/%m/%d"))
```

Comment:
and here for `%x` as well :
```
datetime.datetime.now().strftime("%x") == datetime.datetime.now().strftime("%m/%d/%y")
True

datetime.datetime.now().strftime("%x") == datetime.datetime.now().strftime("%Y/%m/%d")
False
```

Reply:
This is for en_US locale, try change locale and test again. I have added a link to expected behavior of %x and %c in related commit message (fd65809377b47894810fa9e0d4b8262b037d6196). They should represent locale appropriate formats.
Examples from python documentation:
> 08/16/88 (None);
> 08/16/1988 (en_US);
> 16.08.1988 (de_DE)

https://docs.python.org/3.7/library/datetime.html#strftime-and-strptime-behavior

Reply:
@pouyamn thanks I'll review the docs.

Please try to create PRs with smaller changes which focuses on one topic, it's easier to review and also it allows us to pick and choose one fix/feature but not others for example we might go for Glyphs but not this or vice versa.

Reply:
Ok this change make sense but we shouldn't merge it with this PR. Even though the value that we return is wrong, some people might rely on that.

Please create a separate PR for that and we'll bump the major version since it's a breaking change.

Reply:
On the second thought, we can push all the changes in this PR in a major version.

Reply:
> @pouyamn thanks I'll review the docs.
> 
> Please try to create PRs with smaller changes which focuses on one topic, it's easier to review and also it allows us to pick and choose one fix/feature but not others for example we might go for Glyphs but not this or vice versa.

Next time I'll make different branches for PRs, when I work on master, it collects all my changes until the PR is closed.